docs/ref/models/expressions.txt

```diff
@@ -909,6 +909,10 @@ calling the appropriate methods on the wrapped expression.
         Takes a list of expressions and stores them such that
         ``get_source_expressions()`` can return them.
 
+    .. method:: copy()
```

Comment:
I think the `reverse_ordering` documentation could be clarified that it mutates the object rather than returns a copy. It is strange to me that it's different from `QuerySet`'s API. It is out of scope though.